ci/ci/pr.py

```diff
@@ -84,7 +84,32 @@ def try_new_build(source, target):
                         'name': 'batch-test-cloud-sql-config',
                         'readOnly': True
                     }
-                }])
+                }, {
+                    'volume': {
+                        'name': BATCH_TEST_GSA_SECRET_NAME,
+                        'secret': {
+                            'optional': False,
+                            'secretName': BATCH_TEST_GSA_SECRET_NAME
+                        }
+                    },
+                    'volume_mount': {
+                        'mountPath': '/batch-test-gsa-key',
+                        'name': BATCH_TEST_GSA_SECRET_NAME,
+                        'readOnly': True
+                    }
+                }, {
```

Comment:
@danking Do you see what I am forgetting to do to expose the secrets when creating the batch-pod? The yaml for the job looks like this:
```
    volumeMounts:
    - mountPath: /secrets
      name: hail-ci-0-1-service-account-key
      readOnly: true
    - mountPath: /batch-secrets
      name: batch-test-cloud-sql-config
      readOnly: true
    - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
      name: test-svc-token-dl5j4
      readOnly: true

  volumes:
  - name: hail-ci-0-1-service-account-key
    secret:
      defaultMode: 420
      optional: false
      secretName: hail-ci-0-1-service-account-key
  - name: batch-test-cloud-sql-config
    secret:
      defaultMode: 420
      optional: false
      secretName: batch-test-cloud-sql-config
  - name: test-svc-token-dl5j4
    secret:
      defaultMode: 420
      secretName: test-svc-token-dl5j4
```

Reply:
I think I figured it out #5897